src/main/kotlin/com/example/mykku/auth/tool/GoogleOauthClient.kt

```
@@ -0,0 +1,81 @@
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .body(formData)
+                .retrieve()
+                .body<GoogleTokenResponse>()
+                ?: throw MykkuException(ErrorCode.OAUTH_TOKEN_EXCHANGE_FAILED)
+        } catch (e: HttpClientErrorException) {
+            when (e.statusCode.value()) {
+                400 -> throw MykkuException(ErrorCode.OAUTH_INVALID_CODE)
+                401 -> throw MykkuException(ErrorCode.OAUTH_INVALID_CLIENT)
+                else -> throw MykkuException(ErrorCode.OAUTH_TOKEN_EXCHANGE_FAILED)
+            }
+        } catch (e: HttpServerErrorException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)